### DISCUSSION TITLE
ONES的readme，需要一个英文版

### DISCUSSION THREAD

**Discussion by zhuangbiaowei** [kaiyuanshe/ONES#99] [日常讨论]
不知是否已经有相关的翻译计划了？

**Comment by will-ww** [comment]
> 不知是否已经有相关的翻译计划了？

搞呀，志愿者招募中~  :-)

**Comment by wangleo61** [comment]
申请加入哈

发件人: Will ***@***.***>
发送时间: Saturday, May 28, 2022 3:24 PM
收件人: ***@***.***>
抄送: ***@***.***>
主题: Re: [kaiyuanshe/ONES] ONES的readme，需要一个英文版 (Discussion #99)


不知是否已经有相关的翻译计划了？

搞呀，志愿者招募中~ :-)

―
